common/faceting.js

```diff
@@ -851,12 +855,16 @@
                     defer.resolve();
                 }
                 else {
+                    if (scope.facetColumn.hasNullFilter) {
+                        scope.facetModel.appliedFilters.push(facetingUtils.getNullFilter(true));
```

Comment:
Does this mean it should be selected by default when the facetColumn has it? I thought if it was defined for the facetColumn that just means it should be shown..